conda_forge_tick/new_update_versions.py

```diff
@@ -70,35 +73,34 @@ def new_update_upstream_versions(
 
     # Inspection the graph object and node update:
     # print(f"Number of nodes: {len(gx.nodes)}")
-    Node_count = 0
-
+    node_count = 0
     to_update = []
     for node, node_attrs in tqdm.tqdm(_all_nodes):
-        attrs = node_attrs["payload"]
-        if attrs.get("bad") or attrs.get("archived"):
-            attrs["new_version"] = False
-            continue
-        to_update.append((node, attrs))
+        with node_attrs["payload"] as attrs:
+            if attrs.get("bad") or attrs.get("archived"):
+                # "new_version" = False
+                continue
+            to_update.append((node, attrs))
 
-    up_to = {}
     for node, node_attrs in to_update:
         # checking each node
         with node_attrs as attrs:
             up_to = {}
 
-            # verify the actual situation of the package;
-            actual_ver = str(attrs.get("version"))
-
-            # rude exception
+            # avoid
             if node == "ca-policy-lcg":
-                up_to["ca-policy-lcg"] = {"bad": attrs.get("bad"), "new_version": False}
-                Node_count += 1
+                up_to["new_version"] = False
```

Comment:
I'm not certain if this is the most descriptive variable name

Reply:
hha sorry about that, what about `version_data` ?

Reply:
sure